troposphere/static/js/components/Footer.react.js

```diff
@@ -34,7 +34,7 @@ export default React.createClass({
       return (
         <footer className="footer">
           <div className="container">
-            <a href="http://www.jetstream-cloud.org" target="_blank">
+            <a href="http://user.cyverse.org" target="_blank">
```

Comment:
I am going to fix this in quasi-quail and change this to be a `prop` that is passed in from Master.